contents/tutorials/canary-release.md

```diff
@@ -0,0 +1,71 @@
+
+Once you setup PostHog and user identification, you can setup the feature flags for the canary release. Go to “Feature Flags” in your PostHog instance, add a relevant key related to the feature, a description of what you are using the flag for, and a minimal release condition (like only yourself).
+
+![Feature flags](../images/tutorials/canary-release/feature-flag.png)
+
+Add this flag to your code around the feature you want to canary release. Test that it works for you, and that turning it off works as well. Once this goes well, you can begin the process of your canary release.
+
+## The steps to a canary release
+
+If set up correctly, your initial release condition is only yourself. This lets you test a basic implementation before expanding. Beyond releasing to yourself, here is a recommended step-by-step release with conditions for each:
+
+1. **Just yourself:** set email to equal to your own, like `email equals ian@posthog.com`. Test it yourself to make sure the feature flag works and the feature works as expected.
+2. **Internal team:** set email to contain your domain, like `email contains posthog.com` or `email equals <insert multiple team member emails>`. Make sure to communicate with your team about what is being released, so they know to test and look out for issues.
+3. **Beta users, organizations:** set email to contain their domain or set organization to equal theirs, like `email contains twitter.com` or `organization_name equals twitter`. Setting expectations and having a communication channel with them (like a shared Slack channel) is critical for hearing about issues.
+4. **Expanded beta:** set release to a percentage of users or set it to match a popular property. Monitor insights and sessions related to the feature compared to those without. The overall metrics will be more important here.
+5. **Full release:** set release to 100% of users, check metrics again, delete the flag, and announce the feature. 
+
+At PostHog, we do all of these. Our feature flag page contains features at different stages of roll out. For example, when users have specific issues, we might canary release a fix to them before releasing it to others.
+
+![PostHog's feature flags](../images/tutorials/canary-release/posthog-flags.png)
+
+## Monitoring a canary release
+
+The key to a canary release is hearing from the “canary.” Getting feedback and hearing issues is critical to the process whether it is from your team, a limited group of external users, or everyone. Setting clear expectations and having channels for communication enables the canary release to work. Without them, issues go unreported and cause problems for more users than needed.
+
+On top of hearing about issues from users, you can see issues in PostHog. This looks like key metrics or events decreasing, or the number of errors increasing. The benefit of having all your product data in one place is that all your data about your canary release is in one place too.
```

Comment:
```suggestion
On top of hearing about issues from users, you can see issues in PostHog. This may look like key metrics or events decreasing, or the number of errors increasing. The benefit of having all your product data in one place is that all your data about your canary release is in one place too.
```